pages/api/labs/questionGenerators/algebraQuestionGenerator.ts

```diff
@@ -0,0 +1,99 @@
+import { QuestionType } from "../../questionTypes";
+import {
+  getRndInteger,
+  getRndTenthsDecimal,
+  getRndHundredthsDecimal,
+  getRandomItemFromArray,
+} from "../../random";
+import { name } from "../../../api/names";
+export type AlgebraSolveQuestion = {
+  questionType: QuestionType.ALGEBRA_SOLVE_VARIABLE;
+  variableLetter: string;
+  variableproblem: string;
+  algebrasolveModel: AlgebraSolveModel;
+  text: string;
+  answer: string;
+  personname: string;
+};
+
+// Ask coach how to do this properly
+
+
+// Ask coach how to declare 
+export function algebrahelper():list{
+let problemtype = getRandomItemFromArray(["-", "+", "/", "*"]);
+const a = getRndInteger(1,50);
+const b = getRndInteger(1,50);
+const d = a + getRndInteger(1,50);
+const e = getRndInteger(2,10)
+const f = getRndInteger(1,10)
+const g = getRndInteger(2,10)*e
+  if(problemtype == "+") {
+    const problem = [" " + problemtype + " " + a.toString() + " = " + d.toString(),(d-a).toString()];
```

Comment:
Instead of returning an array, I would define a custom type with two properties: question and answer
And then return that object instead.

Reply:
Fixed! Ty